terraform/sponsored_bucket.tf

```diff
@@ -117,3 +117,96 @@ data "aws_iam_policy_document" "sponsored_bucket" {
     }
   }
 }
+
+resource "aws_iam_policy" "sponsored_writers" {
```

Comment:
Maybe it's just me, but I think it would be helpful to move these resources to another TF file, to keep things somewhat organized.

Reply:
I thought the same, but couldn't think of a good name. I'll do my best.